CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Source validation happens before simulation upload and raises an error if no source is present.
 - Cleaner display of `ArrayLike` in docs.
+### Changed
```

Comment:
Just a reminder not to forget to fix the CHANGELOG

Reply:
nice catch, actually develop is messed up too so I just fixed everything in the latest commit.